build.fsx

```diff
@@ -171,6 +171,28 @@ Target "ReleaseGitHub" (fun _ ->
     |> Async.RunSynchronously
 )
 
+// .NET Core SDK and .NET Core
+
+let assertExitCodeZero x = if x = 0 then () else failwithf "Command failed with exit code %i" x
+
+Target "Build.NetCore" (fun _ ->
+    Shell.Exec("dotnet", "restore") |> assertExitCodeZero
+    Shell.Exec("dotnet", "--verbose pack --configuration Release", "src/Argu") |> assertExitCodeZero
+)
+
+Target "RunTests.NetCore" (fun _ ->
+    Shell.Exec("dotnet", "--verbose run --configuration Release", "tests/Argu.NetCore.Tests") |> assertExitCodeZero
+)
+
+let isDotnetSDKInstalled = try Shell.Exec("dotnet", "--version") = 0 with _ -> false
+
+Target "Nuget.AddNetCore" (fun _ ->

```

Comment:
not atm, afaik, it's up to paket team, i'd like an helper cmd too.
That's the minimal flow (maintenance, easy to test)  afaik to add support for .net core.
Also like that you dont need to redefine the dependencies inside paket.template (paket doesnt manage project.json), and are always aligned with `project.json` (used for build)

Are merged **only**:

- `lib/netstandard1.6` (the .net core assemblies)
- the dependencies for `.netstandard1.6`, used only for that profile (no issue with others)

not the metadata like authors, etc, that's defined in paket.template already.